README.md

```diff
@@ -1,11 +1,42 @@
 # Acquacotta
 R Shiny App to display Run Reports generated by the OICR SeqWare pipeline.
+![How it looks](docs/example_look.png)
+
+## Deploying
+
+Before starting Acquacotta, ensure Run Report TSV files are accessible in the directory specified in the [config file](config.yaml). Further options that can be modified are:
+* Port on which Acquacotta is launched
+* Default plots shown to user
+
+### Rscript
+Run the command below and open the listed address (default: http://127.0.0.1:11038) in a Web Broswer:
+```bash
+Rscript app.R
+```
+
+### RStudio
+Load the [app.R file](app.R) in RStudio and click the [Run App button or press Command+Shift+Enter](https://shiny.rstudio.com/tutorial/written-tutorial/lesson1/).
```

Comment:
It might be worth adding a line about how the app may error if packages aren't installed, and give the command for installing packages. (Do you know if there's a one-liner for installing all packages which the program requires?)

Reply:
Instructions added:
https://github.com/oicr-gsi/acquacotta-shiny-run-report/pull/3/commits/dc195816e2fee988637ba3da446026c910115060#L6-L11

Reply:
Thanks :)